src/applications/check-in/locales/en/translation.json

```diff
@@ -303,5 +304,11 @@
   "or-you-can-still-file-your-claim": "Or you can still file your claim within 30 days through the Beneficiary Travel Self-Service System (BTSSS).",
   "claim-submitted_one": "Claim submitted",
   "claim-submitted_other": "Claims submitted",
-  "fpo-header": "Placeholder header"
+  "fpo-header": "Placeholder header",
+  "if-you-traveled-by-bus-train-taxi-or-other-bold--help-text": "<0>If you traveled by bus, train, taxi, or other authorized public transportation, you can’t file a claim in this tool right now.</0> But you can file your claim online, within 30 days, through the Beneficiary Travel Self Service System (BTSSS). Or you can use VA Form 10-3542 to submit a claim by mail, or in person.",
```

Comment:
if these translations where the only thing different is the bold, I think we should just update them everywhere. Double check with UX if it is okay for day-of to get this same update. If not, I would still add the bold to the original translation, but you can choose to not pass a component that makes the text bold. This is so we don't have identical text in our translation file. I believe these strings are already translated.

Reply:
I think this was for two areas and both had updated text as well. Will check with Ben on this though